<launch>

  <node name="vel_commander" pkg="bebop_demo" type="vel_cmd_contr.py" output="screen"/>
  <param name="vel_cmd/update_time" value="0.5" />
  <param name="vel_cmd/sample_time" value="0.01" />
  <param name="max_linear_vel" value="1.5" />

</launch>
